wp-graphql-dad-jokes.php

```diff
@@ -26,6 +26,7 @@ function wp_graphql_dad_jokes_root_query( $fields ) {
 			$get_dad_joke = wp_remote_get('https://icanhazdadjoke.com/', [
 				'headers' => [
 					'Accept' => 'application/json',
+					'User-Agent' => esc_html( get_bloginfo( 'name' ) ),
```

Comment:
Oh, probably don't need to include the user's website name here, instead if you have something like:

```php
'User-Agent' => 'WPGraphQL Dad Jokes (https://github.com/wp-graphql/wp-graphql-dad-jokes)',
```

This way I can see which library people are using to access the API, and if there are any issues with the requests I can bring to your attention here.